uaclient/api/u/pro/security/fix/_common/plan/v1.py

```diff
@@ -799,6 +829,26 @@ def get_pocket_short_name(pocket: str):
         return pocket
 
 
+def _should_update_esm_cache(
+    check_esm_cache: bool,
+    esm_cache_updated: bool,
+    cfg: UAConfig,
+):
```

Comment:
could be explicit that this function returns a `bool`, and add a `return False` after the `if` ends

Reply:
I agree

Reply:
Good catch, I am updating the code